### ISSUE TITLE
Attachment download table still needs some work

### ISSUE THREAD

**Issue by nonprofittechy** [SuffolkLITLab/docassemble-AssemblyLine#65]
The buttons look fine vertically aligned, but without a description the placement of the form name is not quite right.

![image](https://user-images.githubusercontent.com/7645641/112568694-402ba380-8db9-11eb-9eb2-5e3dd8aa0419.png)

1. The name of the file should be top aligned and a bit larger
2. There needs to be something to fill the whitespace--could make the buttons horizontal again as one solution.

There could be other fixes to this page too. It just feels a bit awkward still.

**Comment by purplesky2016** [comment]
The flow on the screen seems a bit awkward. I don't know how difficult it is to do, but something like this would be more natural:

Your form title: blah blah
`View `button
`Make Change` button
`Download `button
`Send` button
`email address` field.

Since they are all left justified, this arrangement would fix the two issues.

**Comment by purplesky2016** [comment]
Here is one version
![image](https://user-images.githubusercontent.com/20231649/116456203-fa3a8300-a82f-11eb-92a5-52961ee25c8a.png)

**Comment by nonprofittechy** [comment]
It is important to add a screenshot here that shows what it looks like with 3-4 forms. The "make changes" button lets you edit all of the forms, not just one.

**Comment by nonprofittechy** [comment]
Decision: for now, just make the "view" button horizontally aligned with the "download" button so it looks normal if you have multiple rows in the table.

**Comment by purplesky2016** [comment]
To prepare for the changes that I'm going to make, I installed  SuffolkLITLab /
docassemble-PetitionToChangeNameOfAdult on my server to see how it looks like right now, but got an error:
`JSONDecodeError: Expecting value: line 1 column 1 (char 0)`

Do I need to install something else? I installed docassemble.CARIRequest, docassemble.ALAffidavitOfIndigency and docassemble.MAPovertyScale (this one is not listed on the include section, I found it by trial and error).

Below is from the error page.

> 

> Needed definition of all_courts at 0.02885s
> Tried to run block at 0.02889s
> 
> (from docassemble.ALMassachusetts:al_massachusetts.yml)
> 
> objects:
>   - trial_court: MACourt
>   - all_courts: MACourtList.using(courts=['housing_courts','bmc','district_courts','superior_courts','land_court', 'juvenile_courts', 'p
> Log
> 
> Traceback (most recent call last):
>   File "/usr/share/docassemble/local3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
>     rv = self.dispatch_request()
>   File "/usr/share/docassemble/local3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
>     return self.view_functions[rule.endpoint](**req.view_args)
>   File "/usr/share/docassemble/local3.8/lib/python3.8/site-packages/docassemble/webapp/server.py", line 7454, in index
>     interview.assemble(user_dict, interview_status, old_user_dict, force_question=special_question)
>   File "/usr/share/docassemble/local3.8/lib/python3.8/site-packages/docassemble/base/parse.py", line 7805, in assemble
>     raise the_error
>   File "/usr/share/docassemble/local3.8/lib/python3.8/site-packages/docassemble/base/parse.py", line 7594, in assemble
>     question_result = self.askfor(missingVariable, user_dict, old_user_dict, interview_status, seeking=interview_status.seeking, follow_mc=follow_mc, seeking_question=seeking_question)
>   File "/usr/share/docassemble/local3.8/lib/python3.8/site-packages/docassemble/base/parse.py", line 8366, in askfor
>     question_result = self.askfor(newMissingVariable, user_dict, old_user_dict, interview_status, variable_stack=variable_stack, questions_tried=questions_tried, seeking=seeking, follow_mc=follow_mc, recursion_depth=recursion_depth, seeking_question=seeking_question)
>   File "/usr/share/docassemble/local3.8/lib/python3.8/site-packages/docassemble/base/parse.py", line 8366, in askfor
>     question_result = self.askfor(newMissingVariable, user_dict, old_user_dict, interview_status, variable_stack=variable_stack, questions_tried=questions_tried, seeking=seeking, follow_mc=follow_mc, recursion_depth=recursion_depth, seeking_question=seeking_question)
>   File "/usr/share/docassemble/local3.8/lib/python3.8/site-packages/docassemble/base/parse.py", line 8037, in askfor
>     exec(command, user_dict)
>   File "<string>", line 1, in <module>
>   File "/usr/share/docassemble/local3.8/lib/python3.8/site-packages/docassemble/base/core.py", line 244, in __init__
>     self.init(*pargs, **kwargs)
>   File "/usr/share/docassemble/local3.8/lib/python3.8/site-packages/docassemble/MACourts/macourts.py", line 292, in init
>     self.load_courts(courts=self.courts,data_path=self.data_path)
>   File "/usr/share/docassemble/local3.8/lib/python3.8/site-packages/docassemble/MACourts/macourts.py", line 383, in load_courts
>     self.load_courts_from_massgov_by_filename(court)
>   File "/usr/share/docassemble/local3.8/lib/python3.8/site-packages/docassemble/MACourts/macourts.py", line 400, in load_courts_from_massgov_by_filename
>     courts = get_courts_from_massgov_url(urls[filename])
>   File "/usr/share/docassemble/local3.8/lib/python3.8/site-packages/docassemble/MACourts/macourts.py", line 25, in get_courts_from_massgov_url
>     jdata = json.loads(jstring)
>   File "/usr/lib/python3.8/json/__init__.py", line 357, in loads
>     return _default_decoder.decode(s)
>   File "/usr/lib/python3.8/json/decoder.py", line 337, in decode
>     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
>   File "/usr/lib/python3.8/json/decoder.py", line 355, in raw_decode
>     raise JSONDecodeError("Expecting value", s, err.value) from None
> json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

**Comment by purplesky2016** [comment]
Ignore this error. Rather than get entangled with other people's interview, I made my own that can generate multiple documents, which is good enough for our purpose.

**Comment by purplesky2016** [comment]
To complete the picture, this is what the current version looks like in mobile:
![image](https://user-images.githubusercontent.com/20231649/117703442-a4eb6380-b197-11eb-997d-f6af00217898.png)

**Comment by purplesky2016** [comment]
I tackled both issues and this is what it would look like:

**Desktop**
![image](https://user-images.githubusercontent.com/20231649/117705016-85553a80-b199-11eb-9f66-8349e5790cb9.png)

**Mobile**
![image](https://user-images.githubusercontent.com/20231649/117704944-71113d80-b199-11eb-9e75-b9ce186741de.png)

Notes:
1. Made the `view` button smaller in mobile mode in order to keep the two buttons still on the same line
2. Moved the `editable copy` button to be above the email address for a better ux.
